3DAmsterdam/Assets/Amsterdam3D/Scripts/SelectionTools/SelectionTools.cs

```diff
@@ -0,0 +1,111 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.Events;
+using LayerSystem;
+using Amsterdam3D.CameraMotion;
+
+namespace Amsterdam3D.Interface
+{
+	// currently works as MVP, still has a bunch of TODOs for better usage.
+
+	//TODO: Move Single click tool to this class, or to a selection tool?
+	public class SelectionTools : MonoBehaviour
+	{
+		[SerializeField]
+		private GameObject canvas;
+		[SerializeField]
+		private SelectionTool selectionTool;
+
+		[SerializeField]
+		private Bounds bounds;
+		private List<Vector3> vertices;
+
+		[SerializeField]
+		private TileHandler tileHandler;
+
+		[SerializeField]
+		private LayerMask buildingLayer;
+
+
+		[SerializeField]
+		private Layer layer;
+
+		private Vector3 groundLevel = new Vector3(0, Constants.ZERO_GROUND_LEVEL_Y, 0);
```

Comment:
misschien beter om ipv Constants.ZERO_GROUND_LEVEL_Y    -Convertcoordinates.Coordconvert.referenceRD.z te gebruiken. dan kunnen we de RD-offset mooi op 1 plaats houden

Reply:
Ahh yes, goeie. Ik pas m' aan